packages/block-editor/src/components/block-tools/zoom-out-toolbar.js

```diff
@@ -124,6 +125,18 @@ export default function ZoomOutToolbar( { clientId, rootClientId } ) {
 			{ canMove && canRemove && (
 				<Shuffle clientId={ clientId } as={ ToolbarButton } />
 			) }
+
+			{ ! isBlockTemplatePart && (
+				<ToolbarButton
+					className="zoom-out-toolbar-button"
+					icon={ edit }
+					label={ __( 'Edit' ) }
+					onClick={ () => {
+						__unstableSetEditorMode( 'edit' );
```

Comment:
Great spot. Strange as I thought that was happening when I tested it. Maybe there's an edge case I missed.

Ping me for a review on your PR.

Reply:
It does visually look like focus is moved. It's just bailing to the body of the frame since it doesn't know where to go, and your next Tab press moves focus into the canvas, which is the right spot. So, it's an easy one to miss.

Reply:
Ready for review, @getdave! https://github.com/WordPress/gutenberg/pull/64725